geoportailv3/static/js/mymaps/mymaps.html

```diff
@@ -50,43 +50,82 @@
 </div>
 
 <!-- Modification Modal -->
-<ngeo-modal ngeo-modal-shown="ctrl.modifying">
+<ngeo-modal ngeo-modal-shown="ctrl.modifying || ctrl.creating">
```

Comment:
@elemoine how do you propose to reuse the same modal form code to both create a new map, as well as modify an existing one. the information if the user wants to create a new map or modify an existing one has to be expressed somehow?